candy-machine/program/src/errors.rs

```diff
@@ -108,4 +108,6 @@ pub enum CandyError {
     NoUnlockWithNFTsStillFrozen,
     #[msg("Setting a sized collection requires the collection metadata to be mutable.")]
     SizedCollectionMetadataMustBeMutable,
+    #[msg("Cannot Switch back from Hidden Settings after items available is greater than 0")]
```

Comment:
We need to update the error message to something like `"Cannot switch from hidden settings"`.

Reply:
@MarkSackerberg one more small change

Reply:
Done.